app/src/main/java/com/kickstarter/libs/UserCurrency.kt

```diff
@@ -0,0 +1,125 @@
+package com.kickstarter.libs
+
+import com.kickstarter.libs.utils.NumberUtils
+import com.kickstarter.models.Project
+import type.CurrencyCode
+import java.math.RoundingMode
+
+class UserCurrency(private val currentConfigType: CurrentConfigType) {
+
+    /**
+     * Returns a currency string appropriate to the user's locale and location relative to a project.
```

Comment:
I am not sure if this is accurate to this class. Might be a case of copy pasta 🕵️‍♀️